src/Sylius/Bundle/ShopBundle/Resources/config/services/twig/component/checkout.xml

```diff
@@ -44,5 +46,19 @@
                 template="@SyliusShop/checkout/address/form/addresses/address/address_book.html.twig"
             />
         </service>
+
+        <service
+            id="sylius_shop.twig.component.checkout.shipping.form"
+            class="Sylius\Bundle\ShopBundle\Twig\Component\Checkout\Shipping\FormComponent"
+        >
+            <argument type="service" id="form.factory" />
+            <argument>Sylius\Bundle\CoreBundle\Form\Type\Checkout\SelectShippingType</argument>
```

Comment:
We should use here a form type from ShopBundle

Reply:
I've fixed it in my PR 😄